src/utils/next.ts

```diff
@@ -53,17 +53,24 @@ export const scaffold = (wrapped : ScaffoldedGetServerSideProps) : GetServerSide
             ctx.z.setTokenData(reqWithSession.session.tokenData);
         }
 
-        const user = await ctx.z.resource('users', 'me').get();
-
         const result = await wrapped(ctx);
 
-        if (hasProps(result)) {
-            const scaffoldedProps : ScaffoldedProps = {
-                ...result.props,
-                user: user.data.data as ZetkinUser,
-            };
-
-            result.props = scaffoldedProps;
+        const resultProps = (user : ZetkinUser | null) => {
```

Comment:
I would rename this function to reflect what it does. Right now it sounds like it would return props (or something like that) but of course it returns nothing, which is fine.

Maybe something like `setUserOnProps()` or `addUserToProps()` or something like that?

Reply:
Another nice option would be something like `augmentProps()` and then we can keep that method and add more things to it, besides the user, down the line.

Reply:
True! I'll rename it to `augmentProps()`